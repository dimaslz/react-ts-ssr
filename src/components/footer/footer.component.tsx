import { Link } from "@/components";

export const Footer = () => (
	<footer className="flex h-16 w-full items-center justify-center space-x-1 p-4 text-sm">
		<span>made with ♥️ by</span>
		<Link
			href="https://twitter.com/home"
			target="_blank"
			rel="noopener noreferrer"
		>
			dimaslz
		</Link>
		<span>from barcelona</span>
		<Link
			href="https://github.com/dimaslz/react-ts-ssr"
			target="_blank"
			rel="noopener noreferrer"
		>
			fork me on github
		</Link>
	</footer>
);
