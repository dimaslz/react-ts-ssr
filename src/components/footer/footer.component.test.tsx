import { render } from "@testing-library/react";

import { Footer } from "./footer.component";

describe("Footer", () => {
	test("should be defined", () => {
		const component = render(<Footer />);

		expect(component).toMatchSnapshot();
	});
});
