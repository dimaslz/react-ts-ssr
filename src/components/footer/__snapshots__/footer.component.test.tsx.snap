// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > should be defined 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="flex h-16 w-full items-center justify-center space-x-1 p-4 text-sm"
      >
        <span>
          made with ♥️ by
        </span>
        <a
          class="text-blue-500 hover:text-blue-700 hover:underline"
          href="https://twitter.com/home"
          rel="noopener noreferrer"
          target="_blank"
        >
          dimaslz
        </a>
        <span>
          from barcelona
        </span>
        <a
          class="text-blue-500 hover:text-blue-700 hover:underline"
          href="https://github.com/dimaslz/react-ts-ssr"
          rel="noopener noreferrer"
          target="_blank"
        >
          fork me on github
        </a>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="flex h-16 w-full items-center justify-center space-x-1 p-4 text-sm"
    >
      <span>
        made with ♥️ by
      </span>
      <a
        class="text-blue-500 hover:text-blue-700 hover:underline"
        href="https://twitter.com/home"
        rel="noopener noreferrer"
        target="_blank"
      >
        dimaslz
      </a>
      <span>
        from barcelona
      </span>
      <a
        class="text-blue-500 hover:text-blue-700 hover:underline"
        href="https://github.com/dimaslz/react-ts-ssr"
        rel="noopener noreferrer"
        target="_blank"
      >
        fork me on github
      </a>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
